Client/qtTeamTalk/channelstree.cpp

```diff
@@ -1489,8 +1494,19 @@ bool ChannelsTree::eventFilter(QObject *object, QEvent *event)
                 {
                     item->setExpanded(true);
                 }
-                return true;
+                else if ((item->type() & USER_TYPE))
+                {
+                    QTreeWidgetItem* parent = item->parent();
+                    int parentid = (parent->data(COLUMN_ITEM, Qt::UserRole).toInt() & ID_MASK);
+                    UserRights rights = TT_GetMyUserRights(ttInst);
+                    if (TT_IsChannelOperator(ttInst, TT_GetMyUserID(ttInst), parentid) ||

```

Comment:
Shouldn't non-operator/admin also be allowed to see if they can transmit?

Reply:
Hmmm, maybe. But text of accessible text shouldn't be different in this case?

Reply:
Doesn't have to be. Does it?

Reply:
hm, maybe it should be "User can transmit..."

Reply:
I see 2 solutions:
- We change text for all case to "User can transmit: yes/no"
- We don't change current text for operators/admins but we change for users by "transmission: allowed/disallowed"